mm/shm/shmat.c

```diff
@@ -80,7 +80,8 @@ static int munmap_shm(FAR struct task_group_s *group,
     {
       /* Free the virtual address space */
 
-      shm_free(group, (FAR void *)shmaddr, region->sr_ds.shm_segsz);
+      vm_release_region(get_group_mm(group), (FAR void *)shmaddr,
```

Comment:
remove the const from line 50 and the cast here

Reply:
done, good catch